deployment/terraform/clear_opensearch/run.sh.tpl

```diff
@@ -0,0 +1,9 @@
+chmod 700 /root/.ssh
+chmod 600 /root/.ssh/id_rsa
+chmod 600 /root/.ssh/config
+chown -R root:root /root/.ssh
+chmod +x /script/clear_opensearch.sh
+echo -e "\nCopy script to Bastion host\n"
+scp /script/clear_opensearch.sh ec2-user@$BASTION_IP:
+echo -e "\nRun cleaning OpenSearch index\n"
```

Comment:
Let's make it plural — **indexes**. In the near future, we will have a few more indexes, so it won't just be one.